TestsCalculator/CalculatorPageTests.cs

```diff
@@ -145,8 +408,8 @@ public void CheckMonthInDropdown()
         {
             //Arrange
             IWebElement monthDropdown = driver.FindElement(By.Id("month"));
-            //Act
-            //object of SelectElement
+
+            //Act & Assert
             SelectElement s = new SelectElement(monthDropdown);
             //get all options
             IList<IWebElement> els = s.Options;
```

Comment:
...get actual values from drop-down:
```suggestion
            IList<string> actualMonths = new List<string>();
            for (int j = 0; j < s.Options.Count; j++)
            {
                    actualMonths.Add(s.Options.ElementAt(j).Text);
            }
```

and assert them:
```
        Assert.AreEqual(expectedMonths, actualMonths);
```